Containerfile

```diff
@@ -11,4 +11,12 @@ RUN Rscript -e "cmdstanr::install_cmdstan()"
 ADD . /.
 
 RUN Rscript -e "pak::pkg_install('local::wweval')"
+RUN Rscript -e "remotes::install_github("CDCgov/ww-inference-model@v0.1.0")
```

Comment:
If this works it's fine, but I'd think you'd want to install these in the reverse order (and/or make `wweval` depend on `ww-inference-model`).

Reply:
`wweval` does depend on `wwinference`, but I still got `wwinference` object not found... this isn't working because of remotes...

Reply:
Ok I think it was just a typo I had!